service/step_defs_test.go

```diff
@@ -43,67 +43,71 @@ import (
-	createVolumeRequest                  *csi.CreateVolumeRequest
-	createRemoteVolumeRequest            *csiext.CreateRemoteVolumeRequest
-	createRemoteVolumeResponse           *csiext.CreateRemoteVolumeResponse
-	createStorageProtectionGroupRequest  *csiext.CreateStorageProtectionGroupRequest
-	createStorageProtectionGroupResponse *csiext.CreateStorageProtectionGroupResponse
-	deleteStorageProtectionGroupRequest  *csiext.DeleteStorageProtectionGroupRequest
-	deleteStorageProtectionGroupResponse *csiext.DeleteStorageProtectionGroupResponse
-	publishVolumeRequest                 *csi.ControllerPublishVolumeRequest
-	unpublishVolumeRequest               *csi.ControllerUnpublishVolumeRequest
-	deleteVolumeRequest                  *csi.DeleteVolumeRequest
-	controllerExpandVolumeRequest        *csi.ControllerExpandVolumeRequest
-	controllerExpandVolumeResponse       *csi.ControllerExpandVolumeResponse
-	controllerGetVolumeRequest           *csi.ControllerGetVolumeRequest
-	controllerGetVolumeResponse          *csi.ControllerGetVolumeResponse
-	listVolumesRequest                   *csi.ListVolumesRequest
-	listVolumesResponse                  *csi.ListVolumesResponse
-	listSnapshotsRequest                 *csi.ListSnapshotsRequest
-	listSnapshotsResponse                *csi.ListSnapshotsResponse
-	listedVolumeIDs                      map[string]bool
-	listVolumesNextTokenCache            string
-	wrongCapacity, wrongStoragePool      bool
-	accessZone                           string
-	capability                           *csi.VolumeCapability
-	capabilities                         []*csi.VolumeCapability
-	nodeStageVolumeRequest               *csi.NodeStageVolumeRequest
-	nodeStageVolumeResponse              *csi.NodeStageVolumeResponse
-	nodeUnstageVolumeRequest             *csi.NodeUnstageVolumeRequest
-	nodeUnstageVolumeResponse            *csi.NodeUnstageVolumeResponse
-	nodePublishVolumeRequest             *csi.NodePublishVolumeRequest
-	nodeUnpublishVolumeRequest           *csi.NodeUnpublishVolumeRequest
-	nodeUnpublishVolumeResponse          *csi.NodeUnpublishVolumeResponse
-	nodeGetVolumeStatsRequest            *csi.NodeGetVolumeStatsRequest
-	nodeGetVolumeStatsResponse           *csi.NodeGetVolumeStatsResponse
-	deleteSnapshotRequest                *csi.DeleteSnapshotRequest
-	deleteSnapshotResponse               *csi.DeleteSnapshotResponse
-	createSnapshotRequest                *csi.CreateSnapshotRequest
-	getReplicationCapabilityRequest      *csiext.GetReplicationCapabilityRequest
-	getReplicationCapabilityResponse     *csiext.GetReplicationCapabilityResponse
-	validateVolumeHostConnectivityResp   *podmon.ValidateVolumeHostConnectivityResponse
-	volumeIDList                         []string
-	snapshotIDList                       []string
-	groupIDList                          []string
-	snapshotIndex                        int
-	rootClientEnabled                    string
+	nGoRoutines                             int
```

Comment:
Only few lines have changed but it shows the entire file has changed. It makes reviewing more tedious.